dGame/dUtilities/VanityUtilities.cpp

```diff
@@ -121,7 +115,7 @@ LWOOBJID VanityUtilities::SpawnSpawner(const VanityObject& object, const VanityO
 	return obj.id;
 }
 
-Entity* VanityUtilities::SpawnObject(const VanityObject& object, const VanityObjectLocation& location) {
```

Comment:
no clue, I was asked to change this to be like this

Reply:
I'm not necessarily opposed, but it'd be good to understand the rationale.

Reply:
These public functions were never actually used in a public manner, and were not designed to, so they shouldn't have been public and likely were just because